app/index.tsx

```diff
@@ -9,20 +9,22 @@ import { ipcRenderer } from 'electron';
 import { createBrowserHistory } from 'history';
 import * as Sentry from '@sentry/react';
 import { captureException, showReportDialog } from '@sentry/react';
+import { _SENTRY_URL } from './constants/sentry';
+
+if (process.env.NODE_ENV === 'production') {
+  Sentry.init({
+    dsn: `${_SENTRY_URL}`
```

Comment:
Can it package if `_SENTRY_URL` is not provided?

Reply:
It can, but it actually had some trouble at runtime. Should be fixed now :)